smartapps/smartthings/logitech-harmony-connect.src/i18n/ar-AE.properties

```diff
@@ -0,0 +1,35 @@
+'''Allows you to integrate your Logitech Harmony account with SmartThings.'''=السماح بدمج حساب Logitech Harmony مع SmartThings.
+'''Connect to your Logitech Harmony device'''=الاتصال بجهاز Logitech Harmony
+'''Logitech Harmony device authorization'''=المصادقة على جهاز Logitech Harmony
+'''Allow Logitech Harmony to control these things...'''=السماح لـ Logitech Harmony بالتحكم بهذه الأجهزة...
+'''Which Switches?'''=أي مفاتيح؟
+'''Which Motion Sensors?'''=أي مستشعرات وجود حركة؟
+'''Which Contact Sensors?'''=أي مستشعرات لمس؟
+'''Which Thermostats?'''=أي أجهزة ثرموستات؟
+'''Which Presence Sensors?'''=أي مستشعرات وجود كائن؟
+'''Which Temperature Sensors?'''=أي مستشعرات قياس درجة الحرارة؟
+'''Which Vibration Sensors?'''=أي مستشعرات اكتشاف اهتزاز؟
+'''Which Water Sensors?'''=أي مستشعرات وجود ماء؟
+'''Which Light Sensors?'''=أي مستشعرات اكتشاف ضوء؟
+'''Which Relative Humidity Sensors?'''=أي مستشعرات قياس الرطوبة النسبية؟
+'''Which Sirens?'''=أي صفارات إنذار؟
+'''Which Locks?'''=أي أقفال؟
+'''You have succesfully logged out of the account.'''=لقد سجلت الخروج بنجاح من الحساب.
+'''Click to enter Harmony Credentials'''=انقر لإدخال بيانات اعتماد Harmony
+'''Note:'''=ملاحظة:
+'''This device has not been officially tested and certified to “Work with SmartThings”. You can connect it to your SmartThings home but performance may vary and we will not be able to provide support or assistance.'''=لم يتم اختبار هذا الجهاز واعتماده رسمياً من أجل ”العمل مع SmartThings“. يمكنك توصيله بمنزلك الذي يعمل مع SmartThings ولكن قد يختلف الأداء ولن نتمكن من توفير الدعم أو المساعدة.
+'''Discovery Started!'''=بدأ الاكتشاف!
+'''Please wait while we discover your Harmony Hubs and Activities. Discovery can take five minutes or more, so sit back and relax! Select your device below once discovered.'''=يرجى الانتظار أثناء اكتشاف موزعات Harmony والأنشطة. قد تستغرق عملية الاكتشاف خمس دقائق أو أكثر، لذلك اجلس واسترخِ! حدد جهازك أدناه فور اكتشافه.
+'''Select Harmony Hubs ({{ numFoundHub }}}}  found)'''=تحديد موزعات Harmony (تم العثور على {{ numFoundHub }}}} (
```

Comment:
why does this one have four end braces `}` ? and there are multiple spaces between end braces and `found`